### DISCUSSION TITLE
Shotcut to a specific page

### DISCUSSION THREAD

**Discussion by robbudge** [frangoteam/FUXA#981] [General]
Our System auto loads the default landing page via 127.0.0.1:1881.
Is it possible to launch directly into a page say 127.0.0.1:1881/maintenance
This  maintenance page would not be on  a standard shortcut, but  would contain a complete Scada system displaying more information and configuration details.

Also it would allow screens in specific locations to directly open the page for that location.

**Comment by unocelli** [comment]
Hi, #514
